dockerfiles/ubuntu/is/Dockerfile

```diff
@@ -16,10 +16,45 @@
 #
 # ------------------------------------------------------------------------
 
-# set base Docker image to AdoptOpenJDK Ubuntu Docker image
-FROM adoptopenjdk:8u222-b10-jdk-hotspot
+# set base Docker image to Ubuntu 20.04 Docker image
+FROM ubuntu:20.04
+
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
-      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.9.0.4"
+      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.9.0.5"
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+#Install JDK Dependencies
```

Comment:
Lets add a space after #. Also end the comment with a full stop. Look at the other places as well.